src/templates/getHandler.ts

```diff
@@ -41,39 +43,48 @@ const makeHandler =
 
     augmentFsModule({ promises, staticManifest, pageRoot, getBase: () => base })
 
-    const NextServer = getNextServer()
-
-    const nextServer = new NextServer({
-      conf,
-      dir: path.resolve(__dirname, app),
-      customServer: false,
-    })
-    const requestHandler = nextServer.getRequestHandler()
-    const server = new Server(async (req, res) => {
-      try {
-        await requestHandler(req, res)
-      } catch (error) {
-        console.error(error)
-        throw new Error('server function error')
+    let bridge
+    const getBridge = (event: HandlerEvent) => {
```

Comment:
```suggestion
    const getBridge = (event: HandlerEvent): Bridge => {
```

? I'm guessing it's possibly not an available type from Vercel

Reply:
Yeah, I meant to look at up. Thanks for the reminder. I would say there must be types as they wetite it in TypeScript, but you never can tell with Vercel.

Reply:
Well, whaddayaknow https://github.com/vercel/vercel/blob/main/packages/node-bridge/bridge.d.ts